Remove Assertion Method fields from ClinVar submission files

northwestwitch opened the pull request:
Fix #3642 - it's mid November!

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>


**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [X] code approved by DNIL
- [x] tests executed by CR

northwestwitch commented:
Main branch:

<img width="998" alt="image" src="https://user-images.githubusercontent.com/28093618/201623908-037830ed-2660-4970-bb4a-105704b57330.png">

This branch:

<img width="677" alt="image" src="https://user-images.githubusercontent.com/28093618/201624256-021afddb-9c12-408e-a3b9-7bbc497bc71b.png">

dnil commented on scout/server/blueprints/clinvar/templates/multistep_add_variant.html:
```diff
@@ -38,7 +38,7 @@
                 After collecting the required fields, this variant gets saved in an open ClinVar submission object.<br>
                 </p>
-                <a href="{{url_for('cases.case',institute_id=institute._id, case_name=case.display_name)}}" class="next action-button" style="text-decoration: none;">Back to Case</a>
+                <a href="{{url_for('cases.case',institute_id=institute._id, case_name=case.display_name)}}" class="previous action-button-previous" style="text-decoration: none;">Back to Case</a>
```
👍

dnil reviewed (approved):
Is this live on ClinVar yet, or was it only pending?

northwestwitch commented:
> Is this live on ClinVar yet, or was it only pending?

Pending. Even if they didn't specify a date in their email, I believe the change will happen on Nov 21st, because of this other email:

```
Thank you for using the ClinVar Submission API!
We are making changes to the API submission "test mode" to allow users to test the new schema before its production release.
The test endpoint for testing submission of data https://submit.ncbi.nlm.nih.gov/apitest/v1/submissions could be unstable during the transition to this new test schema from November 11-13, 2022.
From November 14, 2022, it will only accept the requests adhering to the new upcoming schema.
Key changes in new submission schema:
* We have moved 'assertionCriteria' to top level. Note that only one single assertion criteria (a citation or an electronic document) may be provided for a submission. This assertion criteria will be applied to all interpretations in the submission.
* We have moved 'releaseStatus' to the top level as 'clinvarSubmissionReleaseStatus'. This status will be applied to all the interpretations in the submission.

Please see the new submission schema attached for details.

Note: The main API enpoint https://submit.ncbi.nlm.nih.gov/api/v1/submissions/ still accepts the current schema at https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/#h2_schema_raw.
We will do the production release of this new schema on the main endpoint on November 21, 2022.
If you have any questions, please contact us at [clinvar@ncbi.nlm.nih.gov](mailto:clinvar@ncbi.nlm.nih.gov).
```

I think it's fine to merge. We'd probably release around that date!
